mods/ts/rules/palettes.yaml

```diff
@@ -22,7 +22,6 @@
 		Name: chrome
 		Filename: cameo.pal
 		AllowModifiers: false
-		ShadowIndex: 242
```

Comment:
Indeed that was my plan as well - along with per-widget clock sequences - the whole thing was a PEBKAC.